### PR TITLE
Man pages

### PR DESCRIPTION
**Pull request by simbo1905**
Fixes missing man pages on Ubuntu.

### CODE REVIEW

**Comment by simbo1905** [comment]
Interestingly my travis got further with this and failed with the issue: 

> Yum-utils package has been deprecated, use dnf instead.\nSee 'man yum2dnf' for more information."

**Comment by simbo1905** [comment]
I made the yum2dnf change and it causes fedora git to get a lot further but it eventually dies with  https://travis-ci.org/sobolevn/git-secret/jobs/389884422#L3236 `sudo: /usr/local/bin/rspec: command not found`

**Comment by joshrabinowitz** [comment]
Looking at that output, I think rspec might be at /home/travis/.rvm/gems/ruby-2.4.1/bin/rspec

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,12 @@
 matrix:
   fast_finish: true
   include:
+    - os: linux
+      env: KITCHEN_REGEXP="gnupg-git-fedora-latest"
+      services: docker
+      sudo: required
+      language: ruby
+      rvm: 2.4
```
Comment (simbo1905) [inline .travis.yml]: here i am just moving the slowest test to the top to kick off first on travis.

**Comment by joshrabinowitz** [comment]
LGTM

**Comment by sobolevn** [comment]
@simbo1905 awesome! thanks!

**Pull request merged**